### ISSUE TITLE
`Property name kube-dns not found`

### ISSUE THREAD

**Issue by kamilgregorczyk** [kubernetes/minikube#3642]
Whenever I try to enable kube-dns with `minikune addons enable kube-dns` I get `Property name kube-dns not found` eventhough this addon is listed:
```
➜  ~ minikube addons list
- addon-manager: enabled
- dashboard: enabled
- default-storageclass: enabled
- efk: disabled
- freshpod: disabled
- gvisor: disabled
- heapster: enabled
- ingress: disabled
- kube-dns: disabled
- metrics-server: disabled
- nvidia-driver-installer: disabled
- nvidia-gpu-device-plugin: disabled
- registry: disabled
- registry-creds: disabled
- storage-provisioner: enabled
- storage-provisioner-gluster: disabled
```
**Environment**:

**Minikube version** (use `minikube version`): v0.33.1
- **OS** MacOS 14.4.3
- **VM Driver**: virtualbox
- **ISO version**: `   "Boot2DockerURL": "file:///Users/kamilgregorczyk/.minikube/cache/iso/minikube-v0.33.1.iso",

**Comment by husamuddin** [comment]
I'm having the same issue

**Environment**:
- **OS** Archlinux
- **Minikube**: v0.33.1
- **VM Driver**: virtualbox

**Comment by tstromberg** [comment]
I suspect this is fallout from kube-dns being deprecated in favor of CoreDNS in Kubernetes. We'll have to take a closer look, but I suspect this addon is no longer meaningful.

**Comment by hpandeycodeit** [comment]
@tstromberg I can work on this one. 
Are we looking to remove the kube-dns addon completely?

**Comment by tstromberg** [comment]
@hpandeycodeit - Yes, I don't think it makes sense to exist any longer, to be honest.

**Comment by hkroger** [comment]
Is there a workaround to get kubernetes DNS working somehow? I am facing this issue and I also think that because of this might be that the pods cannot access any DNS either. I see that a service cannot start because it cannot resolve some non-kubernetes hostname. Can it be because of this?

**Comment by kamilgregorczyk** [comment]
I don't think so, coredns that automatically boots with minikube works
flawlessly for me. I just wanted to try out kube DNS

śr., 13 lut 2019, 11:49: Hannu Kröger <notifications@github.com> napisał(a):

**Comment by hkroger** [comment]
Actually my problem is that coredns does not start because it gets into resolving loop because systemd-resolved has 127.0.0.53 in resolv.conf. Should I create a ticket about this?

**Comment by hpandeycodeit** [comment]
@tstromberg After I build the minikube from the source, I don't see any **kube-dns** addon in the list:

`
out/minikube addons list
- addon-manager: enabled
- dashboard: disabled
- default-storageclass: enabled
- efk: disabled
- freshpod: disabled
- gvisor: disabled
- heapster: disabled
- ingress: disabled
- logviewer: disabled
- metrics-server: disabled
- nvidia-driver-installer: disabled
- nvidia-gpu-device-plugin: disabled
- registry: enabled
- registry-creds: disabled
- storage-provisioner: enabled
- storage-provisioner-gluster: disabled
` 
Looks like the code is already merged but not yet released?

**Comment by tioxy** [comment]
The version v0.33.1 had kube-dns as an [addon](https://github.com/kubernetes/minikube/blob/v0.33.1/pkg/minikube/assets/addons.go#L114).

As @hpandeycodeit pointed, it's already merged #3556

Updating your minikube version to >= 0.34.0 should fix it.

**Comment by tstromberg** [comment]
Marking as obsolete, as the `kube-dns` addon was removed in a previous release. Please re-open if you run into this again.
